### PR TITLE
Add cookie consent banner

### PR DESCRIPTION
**Pull request by NohYujeong**

### CODE REVIEW

File: app/components/popupConsentBanner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            Privacy Policy.
+          </a>
+        </div>
+      </div>
+      <div className={s.bannerButton}>
+        <Button
+          elementType="button"
+          onClick={() => {
+            setAcceptCookie('true');
+            Cookies.set(ACCEPT_COOKIE_KEY, 'true');
```
Comment (TylorShin) [inline app/components/popupConsentBanner/index.tsx]: if you don't specify when the cookie should be expired, it sets the default expires behavior.
```
Default: Cookie is removed when the user closes the browser.
```
image

I suggest setting the expiring time as long as possible.

File: app/components/popupConsentBanner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const useStyles = require('isomorphic-style-loader/useStyles');
+const s = require('./popupConsentBanner.scss');
+
+const ACCEPT_COOKIE_KEY = 'a_c';
+
+type AcceptAnswer = 'true' | 'false';
+
+const PopupConsentBanner: React.FC = () => {
+  useStyles(s);
+  const [acceptCookie, setAcceptCookie] = useState<AcceptAnswer>('true');
```
Comment (TylorShin) [inline app/components/popupConsentBanner/index.tsx]: you can omit the below useEffect by

```
const [acceptCookie, setAcceptCookie] = useState<AcceptAnswer>(Cookies.get(ACCEPT_COOKIE_KEY) || 'false'); 
```

File: app/components/popupConsentBanner/popupConsentBanner.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+.link {
+  text-decoration: underline;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+@media (max-width: $mobile_width) {
```
Comment (TylorShin) [inline app/components/popupConsentBanner/popupConsentBanner.scss]: 👍

File: app/routes.tsx [hunk 4]
```diff
@@ -282,6 +283,7 @@ const RootRoutes: React.FC<RootRoutesProps> = props => {
       <DialogComponent />
       <FeedbackButton />
       <SnackbarComponent />
+      <PopupConsentBanner />
```
Comment (TylorShin) [inline app/routes.tsx]: I think that wrapping it with <NoSSR> will make the component safer.

Comment (NohYujeong) [reply app/routes.tsx]: I don't understand `wapping it`. Is that mean,
```
<div className={s.wrapperStyle}>
  <PopupConsentBanner />
</div>
```
just this, or other mean?

Comment (TylorShin) [reply app/routes.tsx]: NoSSR. I guess Github system omited \<\> for some reason.

**Pull request merged**